packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -65,11 +63,8 @@ export function WithdrawalCountdown({
     parentChainId: tx.parentChainId
   })
 
-  const remainingTextOrEmpty =
-    isLargeScreen && minutesLeft > 0 ? ' remaining' : ''
-
   const timeLeftText =
     minutesLeft === 0 ? 'Almost there...' : dayjs().to(txConfirmationDate, true)
 
-  return <span>{timeLeftText + remainingTextOrEmpty}</span>
+  return <span>{timeLeftText}</span>
```

Comment:
Same, might need to check it's impact on E2E

Reply:
same as above